From 72bfefede6ab04cdbea914cd550229c1e160117b Mon Sep 17 00:00:00 2001
From: Julia Lawall <julia.lawall@lip6.fr>
Date: Tue, 14 Feb 2017 07:19:36 +0100
Subject: [PATCH 2206/3082] drm/amd/dc: hw_sequencer: fix semicolon.cocci
 warnings

Remove unneeded semicolon.

Generated by: scripts/coccinelle/misc/semicolon.cocci

Signed-off-by: Julia Lawall <julia.lawall@lip6.fr>
Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Reviewed-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/dce110/dce110_hw_sequencer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/dce110/dce110_hw_sequencer.c b/drivers/gpu/drm/amd/display/dc/dce110/dce110_hw_sequencer.c
index 0afb3c6..9b6d251a 100644
--- a/drivers/gpu/drm/amd/display/dc/dce110/dce110_hw_sequencer.c
+++ b/drivers/gpu/drm/amd/display/dc/dce110/dce110_hw_sequencer.c
@@ -2070,7 +2070,7 @@ static void init_hw(struct core_dc *dc)
 			true);
 	}
 
-	dce_clock_gating_power_up(dc->hwseq, false);;
+	dce_clock_gating_power_up(dc->hwseq, false);
 	/***************************************/
 
 	for (i = 0; i < dc->link_count; i++) {
-- 
2.7.4

